static/app/api.tsx

```diff
@@ -282,12 +281,8 @@ export class Client {
         return;
       }
 
-      if (isUndefined(func)) {
-        return;
-      }
-
       // Call success callback
-      return func.apply(req, args); // eslint-disable-line
+      return func?.apply(req, args); // eslint-disable-line
```

Comment:
could write `return undefined;` in the lines above and remove this eslint-disable.

theme of the day!